site/en/guide/tensors.md

```diff
@@ -269,7 +269,8 @@ print(tensor.eval())
 ```
 
 The `eval` method only works when a default `tf.Session` is active (see
-Graphs and Sessions for more information).
+[Graphs and Sessions](https://www.tensorflow.org/guide/graphs) for more 
```

Comment:
```suggestion
 [Graphs and Sessions](./graphs.md) for more 
```

Reply:
To make sure the docs are browseable in GitHub, use relative path (with extension) for links to guides/tutorials.
The extension is removed when these docs are imported and pushed to tensorflow.org (as well as some other transformations).

I need to update the [style guide](https://www.tensorflow.org/community/documentation#links_in_markdown)